src/laser/Laser.H

```diff
@@ -22,6 +22,7 @@ public:
     std::string m_name {""};
     amrex::Real m_a0 {0.}; /**< Laser peak normalized amplitude */
     amrex::Real m_w0 {0.}; /**< Laser waist */
+    amrex::Real m_init_angle_yz {0.}; /**< initial angle of laser pulse fron tilt on yoz */
```

Comment:
The name could be more explicit, and the description is not too accurate. Here is a suggestion. Note that renaming the variable here would require other changes below.
```suggestion
    amrex::Real m_propagation_angle_yz {0.}; /**< Propagation angle of the pulse in the yz plane (0 is the along the z axis) */
```